ods_ci/tests/Tests/400__ods_dashboard/415__ods_dashboard_projects/415__ods_dashboard_projects.robot

```diff
@@ -15,9 +16,12 @@ Test Teardown      Close All Browsers
 *** Variables ***
 ${PRJ_TITLE}=   ODS-CI DS Project
 ${PRJ_TITLE1}=    ODS-CI DS Project1
+${PRJ_TITLE2}=    ODS-CI DS Project2
+${PRJ_TITLE3}=    =    ODS-CI DS Project3
```

Comment:
why adding 3 new projects? You can have 1 project for all the test you're adding, or using the same project as the others, i.e., ${PRJ_TITLE}